app/templates/components/app-header.hbs

```diff
@@ -1,11 +1,11 @@
-<div class='logo-container'>
+<div class="logo-container">
   {{#link-to indexRoute}}
     {{#if hasLogo}}
-      <img src='{{logo}}' />
+      <img src="{{logo}}" />
     {{else if isTopbarStyle}}
-      <img src='assets/images/air-1-white.png' />
+      <img src="assets/images/air-1-white.png" />
```

Comment:
air-1-white? to be updated with one of the new login. i.e. the one you preset in every applications